### DISCUSSION TITLE
When to rebuild chapters with decrypt?

### DISCUSSION THREAD

**Discussion by vwkd** [mkb79/audible-cli#168] [Q&A] [answered]
The `decrypt` subcommand has a `-r, --rebuild-chapters` option to "Rebuild chapters with chapters from voucher or chapter file".

When should this option be used? When is it necessary over using the chapters that are present by default?

**Comment by mkb79** [comment, marked as answer]
The chapter data in the aax or aaxc container file does not contain any chapter names. They are usually named `Chapter [number]` or for German audiobooks `Kapitel [number]`. If you want nearly the same chapter information like the Audible Android or iOS app shows, you can use the `-r` option. This will use a previous downloaded chapter file which contain the same chapter information like the origin app.

**Reply by vwkd** [reply]
Thanks! Does this change only the chapter names or does it add more information?

**Reply by mkb79** [reply]
It will replace the chapter information with the one from the chapter file. It will only work, if the chapter count is the same. So theoretically it is possible, that the chapter times can be different.
